README.md

```diff
@@ -1,24 +1,606 @@
-# `pg_net`
+<h1 style= 'border-bottom: 0;'>PG_NET</h1>
+
+<em>a PostgreSQL extension that enables asynchronous (non-blocking) HTTP/HTTPS requests with SQL</em>
 
 <p>
+    <a href="">
+        <img src="https://img.shields.io/badge/postgresql-12+-blue.svg" alt="PostgreSQL version" height="18">
+    </a>
+    <a href="https://github.com/supabase/pg_net/blob/master/LICENSE">
+        <img src="https://img.shields.io/pypi/l/markdown-subtemplate.svg" alt="License" height="18">
+    </a>
+    <a href="https://github.com/supabase/pg_net/actions">
+        <img src="https://github.com/supabase/pg_net/actions/workflows/main.yml/badge.svg" alt="Tests" height="18">
+    </a>
+</p>
+
+<strong>Documentation</strong>:<a href="https://supabase.github.io/pg_net" target="_blank">https://supabase.github.io/pg_net</a>
+
+<hr />
+<h1>Contents</h1>
```

Comment:
@TheOtherBrian1 Awesome PR!

Any reason on why not use markdown headings here? e.g.

```suggestion
# Contents
```

Reply:
Seems we could avoid some HTML tags in other part too.

Reply:
@steve-chavez When I began editing the file, I noticed that the majority of the document was written in HTML. For consistency purposes, I decided to prioritize HTML over markdown. If it is preferable, I could change it all to Markdown.

Reply:
>  If it is preferable, I could change it all to Markdown.

That would be great. Thanks!

Reply:
@steve-chavez It's all in markdown now